packages/dd-trace/src/instrumenter.js

```diff
@@ -39,6 +39,8 @@ class Instrumenter {
   enable (config) {
     config = config || {}
 
+    this._enabled = true
```

Comment:
why move this here?

Reply:
Already loaded modules would not be tentatively patched otherwise (basically the next line).

Reply:
so... maybe worth a comment on why we need to do it as soon as possible? ;)

Reply:
It makes sense to start with this, since otherwise of course calling any other method might fail since it's not effectively enabled yet. The previous behavior is the one that would have needed an explanation, but it was wrong anyway.

Reply:
and we have regression test for it.